src/components/common/MainButton/MainButton.jsx

```diff
@@ -0,0 +1,12 @@
+import styles from './MianButton.module.scss';
+
+const MainButton = ({ btnStyle, children }) => {
+  return (
+    <button className={`${styles.button} ${styles[btnStyle]}`}>
```

Comment:
Add type for the button.

Make it possible to pass through props, but let the default value to be "button"

` ({ btnStyle, children, type = 'button' }) 
type={type}
`

Reply:
I think, prop types wont be redundant either)